api/python/quilt3/packages.py

```diff
@@ -1387,19 +1412,19 @@ def physical_key_is_temp_file(pk):
         for lk in temp_file_logical_keys:
             self._set(lk, pkg[lk])
 
-        top_hash = pkg._build(name, registry=registry, message=message)
+        pkg._push_manifest(name, registry, top_hash)
 
         if print_info:
             shorthash = registry.shorten_top_hash(name, top_hash)
-            print(f"Package {name}@{shorthash} pushed to s3://{dest_parsed.bucket}")
+            print(f"Package {name}@{shorthash} pushed to s3://{registry.base.bucket}")
 
             if user_is_configured_to_custom_stack():
                 navigator_url = get_from_config("navigator_url")
 
                 print(f"Successfully pushed the new package to "
-                      f"{catalog_package_url(navigator_url, dest_parsed.bucket, name, tree=False)}")
+                      f"{catalog_package_url(navigator_url, registry.base.bucket, name, tree=False)}")
             else:
-                dest_s3_url = str(dest_parsed)
+                dest_s3_url = str(registry.base)  # FIXME
```

Comment:
Not sure what should be printed here.

Reply:
Well the package still has a destination, so we should print the value of registry=, which I think is what you have?